### ISSUE TITLE
Official very important must-read great prose thingamajig - a.k.a. info from the devs

### ISSUE THREAD

**Issue by JakobTischler** [Courseplay/courseplay#1133]
Dear Ladies and Lads,

as you might have noticed, progress and activity has become rather stale here recently. This is due to all three of us (Thomas, Claus and me) being rather busy with other projects, and - more importantly - real life. Work and family have the priority here. At the same time, there's a bit of low motivation at the moment. This is due to no one in particular, but rather a constant stream of requests and re-requests and changing one thing for someone and thus breaking it for someone else, etc. pp.

With all that in mind, we'd like to inform you that we put the project on *"break"* status. Meaning: we take a break. Even more meaning: the project isn't dead, but we won't be responding to your posts. We might work on something different. We might work on something within CP that we weren't able to because of the feature and fix requests. We might not do anything at all.

This won't be permanent, but of course there'll be no *"final vacation date"* (just like there's never a future release date, for obvious reasons).

We'd like to thank everyone who has donated, but encourage you to stop donations for now, as us receiving money while we don't do anything wouldn't be fair. We have also no problem with you posting questions that *other users* might be able to answer.

Cheerio,
*-- The Management*

**Comment by sootysax** [comment]
thanks for all your hard work..  i think you leave it in a very playable state, and when the new patch and script docs are released, im sure we will see you reinspired to continue development.

**Comment by mereman** [comment]
I agree with sootysax I'm having very few problems and am very happy with CP as it stands now. Just take a good long break and come back refreshed.

It's the one mod I couldn't do without, Thanks for your work guys.

**Comment by Mopar** [comment]
Couldn't have said it better then what sootysax/mereman said. have I found bugs/issues since the 101 release? yea, but I dont know if they are CP issues or how the other mods are made that don't follow "the norm", but I have found ways around some of those issues, which add to the fun! It is those reasons why I didn't say anything, because of the sense of accomplishment to find a way to make something work.

This mod is great, and very, very, very helpful, especially on maps like PV15 where it would be very difficult to do anything by yourself without it, and to be honest, since I started using CP, I just can't imagine playing without it.

TL:DR... Endless thanks for your guys/gals work and await your return!

**Comment by Malaras** [comment]
Altho farm sim patch 1.3 prolly be reasled in 1-3 weeks. Hope it don't break anything..

Have a good break.

**Comment by Karlshoj** [comment]
Not a lot to add to what has already been said. Just want to say thank you so much for this "can´t live without" mod. Without CP I simply would´nt play FS.

Enjoy your break

**Comment by rock5046** [comment]
Anyway you can re-enable multiplayer?  I think a lot of folks would accept it in whatever condition that it is in.  Thanks for all your hard work!

**Comment by Tanglewolf** [comment]
thanks for everything you have done so far guys, this is 'the mod' for me :)

**Comment by kwguk** [comment]
Yep what Tanglewolf said.

Thanks guys for all your efforts

**Comment by Jebadiah04** [comment]
Just wanted to throw in there. Without courseplay I wouldn't play fs15.  I have a very enjoyable experience trying to get as much machinery moving smoothly as possible.  Sometimes I have so many things going at once that when I close the game down I don't wanna reopen it and figure out what all my stuff was doing. Hahah. So much fun.

**Comment by Satissis** [comment]
Bumping this to top

**Comment by erksartori** [comment]
You know what I think about this? I think we all deserve a descaço because a work of this size deserves much more than that. It is a very, very deserved, parabens work. and of course you guys deserve a lot more than that.

**Comment by Satissis** [comment]
#1346
